.pre-commit-config.yaml

```diff
@@ -0,0 +1,25 @@
+# See https://pre-commit.com for more information
+repos:
+  - repo: https://github.com/PyCQA/flake8
+    rev: 6.1.0
+    hooks:
+      - id: flake8
+        # Required to make flake8 read from pyproject.toml for now :(
+        additional_dependencies: ["flake8-pyproject"]
+  - repo: https://github.com/psf/black
+    rev: 23.9.1
+    hooks:
+      - id: black
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v1.6.0
+    hooks:
+      - id: mypy
+        language_version: "3.10"
+        args: [--explicit-package-bases,--config-file,pyproject.toml]
+        additional_dependencies:
+          [
+            "types-requests",
+            "sqlalchemy-stubs",
+            "types-pyYAML",
+            "types-pkg-resources",
+          ]
```

Comment:
```suggestion
        additional_dependencies:
          - "sqlalchemy-stubs"
          - "types-pkg-resources"
          - "types-pyyaml"
          - "types-requests"
          - "types-setuptools"
        exclude: "build|broken-plugins/setup.py"
```

Reply:
I think the reason `exclude` doesn't work in the pyproject.toml is that pre-commit is the one finding the files and passing them to mypy. If you exclude them via pre-commit it seems to work.

Reply:
Oh right, make sense!